### ISSUE TITLE
Remove all "...", ">", ">>>", "$" etc. in docs

### ISSUE THREAD

**Issue by kevinlacaille** [planetlabs/planet-client-python#338]
By removing all `...`, `>`, `>>>`, `$`, etc. in the examples/code snippets in the docs, the aforementioned code will be easily useable without having to then remove all the unusable characters afterwards.

Docs that need changing: 
- [README.md](https://github.com/planetlabs/planet-client-python/blob/v2/README.md)
- [guide.md](https://github.com/planetlabs/planet-client-python/blob/v2-polling-output-310/docs/guide.md)
- [index.md](https://github.com/planetlabs/planet-client-python/blob/v2-polling-output-310/docs/index.md)
- [examples/README.md](https://github.com/planetlabs/planet-client-python/tree/v2-polling-output-310/examples)
- [CONTRIBUTING.md](https://github.com/planetlabs/planet-client-python/blob/v2/CONTRIBUTING.md)
- In any comments in code itself that I've missed

For example, change this:

```python
>>> import asyncio
>>> import os
>>> from planet import Session
>>>
>>> async def main():
...     async with Session() as sess:
...         # perform operations here
...         pass
...
>>> asyncio.run(main())

```

to this:
```python
import asyncio
import os
from planet import Session

async def main():
     async with Session() as sess:
         # perform operations here
         pass

 asyncio.run(main())

```

**Comment by kevinlacaille** [comment]
Ah, looks like this is a duplicate of #275 and it seems to be blocked by our doc test (see [this comment](https://github.com/planetlabs/planet-client-python/pull/271#discussion_r617731380) in #271). We'll leave it open, as it provides more context.

**Comment by JuLeeAtPlanet** [comment]
This has been addressed in examples throughout the docs.
